lib/types/host-rules.ts

```diff
@@ -14,4 +14,5 @@ export interface HostRule {
   enableHttp2?: boolean;
   concurrentRequestLimit?: number;
   resolvedHost?: string;
+  githubTokenWarn?: boolean;
```

Comment:
I don't think it should/needs to be inside hostRules. Can just be a top level config option

Reply:
Thanks for the feedback! How would you suggest getting access to that top level config in `getChangeLogJSON`?

Reply:
It can be part of global config and therefore accessible by getting the global config object

Reply:
I noticed it wasn't on `config` but looks like I can just import it. Neat. Thanks!